src/garage/torch/value_functions/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""Baselines (value functions) which use NumPy as a numerical backend."""
```

Comment:
This docstring should read `"""Value Function which use PyTorch."""` (Or something)

Reply:
I missed this one. Thanks